tgui/packages/tgui/interfaces/CrewMonitor.js

```diff
@@ -0,0 +1,106 @@
+import { sortBy } from 'common/collections';
+import { useBackend } from "../backend";
+import { Window } from "../layouts";
+import { NanoMap, Box, Table, Tooltip, Icon, Button } from "../components";
+import { TableCell } from '../components/Table';
+
+export const CrewMonitor = (props, context) => {
+  const { act, data } = useBackend(context);
+  const crew = sortBy(
+    crewmember => crewmember.name,
+  )(data.crewmembers || []);
+  return (
+    <Window>
+      <Window.Content>
+        {crew.map(crewmember => (
+          (crewmember.sensor_type === 3 ? (
+            <Box
+              name={crewmember.sensor_type === 3 ? "circle" : "none"}
+              position="absolute"
+              className="NanoMap_Marker"
+              top={(((255 - crewmember.y) * 2) + 2) + 'px'}
+              left={((crewmember.x * 2) + 2) + 'px'}
+              tooltip={crewmember.name}>
+              <Icon
+                name="circle"
+                color={crewmember.dead ? 'red' : 'green'}
+                size={0.5}
+              />
+              <Tooltip content={crewmember.name} />
+            </Box>
+          ) : (null))
+        ))}
+        <NanoMap />
+        <div className="NanoMap_Content_Offset">
```

Comment:
You should be able to do `<Box as="div" classname="NanoMap_Content_Offset">` here, instead of a normal `<div>`. Just to keep consistent with using TGUI props.

Reply:
![image](https://user-images.githubusercontent.com/25063394/84200111-1f9a9300-aa9e-11ea-89fa-f4934d6fd035.png)
This happens when I do that

Reply:
probably because it's `className` not `classname`
also the default rendering of `Box` is a div, you don't need to do as div